packages/api-client-core/src/graphql-live-query-utils/index.ts

```diff
@@ -0,0 +1,5 @@
+import { createApplyLiveQueryPatch } from "./createApplyLivequeryPatch.js";
+import { applyJSONDiffPatch } from "@n1ru4l/graphql-live-query-patch-jsondiffpatch";
+
+export const applyLiveQueryJSONDiffPatch =
+    createApplyLiveQueryPatch(applyJSONDiffPatch);
```

Comment:
Can we import some of the tests from the original package and test the imported code here?